meta-psa/recipes-security/psa-trusted-storage-linux/psa-trusted-storage-linux_1.0.0.bb

```diff
@@ -8,21 +8,31 @@ HOMEPAGE = "https://github.com/ARMmbed/psa_trusted_storage_linux.git"
 DESCRIPTION = "A Linux C library reference implementation of the PSA protected_storage.h API"
 SECTION = "libs"
 
+inherit systemd
+
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = " \
     file://LICENSE;md5=302d50a6369f5f22efdb674db908167a \
     file://apache-2.0.txt;md5=3b83ef96387f14655fc854ddc3c6bd57 \
     "
 
-SRC_URI = "git://git@github.com/armmbed/psa_trusted_storage_linux.git;protocol=ssh;nobranch=1"
+SRC_URI = " \
+    git://git@github.com/armmbed/psa_trusted_storage_linux.git;protocol=ssh;nobranch=1 \
+    file://psa-ecryptfs.service \
+    file://psa-ecryptfs-init.sh \
+    "
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 SRCREV = "2be73f8a50eca10b45f952798e2100afd00a99f2"
 
-PACKAGES =+ "${PN}-example"
+PACKAGES =+ "${PN}-test"
 
 PV .= "+git${SRCPV}"
 S = "${WORKDIR}/git"
 
-FILES_${PN}-example = "${bindir}"
+FILES_${PN}-test = "${bindir}"
+
+RDEPENDS_${PN}-test += "${PN}"
```

Comment:
This dependency is not right, I'm afraid. The main package installs the systemd service file which calls the shell script, but the shell script is provided by the test package. I think we should have `RDEPENDS_${PN} += "${PN}-test"` instead, but at the same time the main package having a runtime dependency on the test package doesn't feel right to me.
Is this shell script used only for testing purposes?

Reply:
Yeah, if e.g. PDMC were to use PSA storage, it would need the shell script, right? So I think the shell script should be part of the default package. Then the dependency here would make sense.